### ISSUE TITLE
Support multiple dynamic blocks

### ISSUE THREAD

**Issue by SvenKirschbaum** [fgouteroux/terraform-provider-mimir#24]
Hello!

thanks for developing this provider!

While trying to create an Alertmanager configuration I noticed that it is currently not possible to create multiple email_configs for a single receiver, or multiple time_intervals blocks per time_interval, although both would be perfectly valid according to the alertmanager config spec. By looking at the code i determined that there are multiple other types with similar restrictions.

I was wondering if there is a reason for this, seemingly arbitrary, limitation? If not please consider this a feature request. I can look into providing a corresponding Pull Request as well.

Thanks in advance!

**Comment by fgouteroux** [comment]
hi @SvenKirschbaum ,

You are right, we can define multiple block inside receivers, I don't remember why I limit to one item in the list (MaxItems: 1). Maybe a trailing c/p. I will have a look to other block concerned but now I'm wondering if I should rename the block name suffix to remove the plurial word as we could define multiple block.

Exemple with pagerduty receiver config: `pagerduty_configs` should be named `pagerduty_config`
```
    resource "mimir_alertmanager_config" "mytenant" {
      route {
        group_by = ["..."]
        group_wait = "30s"
        group_interval = "5m"
        repeat_interval = "1h"
        receiver = "pagerduty"
      }
      receiver {
        name = "pagerduty"
        pagerduty_configs {
          routing_key = "secret"
          details = {
            environment = "dev"
          }
        }
        pagerduty_configs {
          routing_key = "secret2"
          details = {
            environment = "dev2"
          }
        }
      }
    }
```

**Comment by fgouteroux** [comment]
Fixed in release [0.2.5](https://github.com/fgouteroux/terraform-provider-mimir/releases/tag/v0.2.5)
